### PR TITLE
KHADRI-210924-0711-PM - SuperMarket

### PR DESCRIPTION
**Pull request by roohishaik128**
Updated SuperMarket

### CODE REVIEW

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 1]
```diff
@@ -19,24 +19,20 @@
...
 	private Scanner scanner;
 
-	private EmployeeDao empDao;;
-
+	private EmployeeDao empDao;
 	private CustomerDao custDao;
-
 	private MovieDao movieDao;
 	private MobileDao mobileDao;
 	private SuperMarketDao superMarketDao;
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: why you need superMarketDao in the Driver class? move to processor.

File: src/main/java/com/khadri/jdbc/prepare/data/supermarket/dao/SuperMarketDao.java [hunk 2]
```diff
@@ -12,12 +12,12 @@
 public class SuperMarketDao {
 
 	Connection conn = JdbcConnectionUtil.getConnection();
-	PreparedStatement pstmt;
+	PreparedStatement pstmt ;
 	Statement stmt;
-
+	
 	public void insertSuperMarketData(SuperMarket market) {
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/supermarket/dao/SuperMarketDao.java:18]: How does you know inserted record successfully?

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 1 above)
Comment (roohishaik128) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: SIR MORNING YOU ARE COMMENT IN MOVIE MODULE  AND TELL ALSO CHANGE EVERYONE

File: src/main/java/com/khadri/jdbc/prepare/data/supermarket/dao/SuperMarketDao.java (hunk 2 above)
Comment (roohishaik128) [reply src/main/java/com/khadri/jdbc/prepare/data/supermarket/dao/SuperMarketDao.java:18]: YES SIR IT'S INSERTED SUCCESSFULY

**Review by thasmiyas39 — APPROVED** [review]

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**